dev/com.ibm.ws.jsfContainer/bnd.bnd

```diff
@@ -19,7 +19,7 @@ Private-Package: \
     com.ibm.ws.jsf.container.*
 
 # Ensure we do NOT import JSF spec API from server-side
-# TODO: we should find a way to avoid importing Weld packages directly 
+# TODO: we should find a way to avoid importing Weld packages directly
```

Comment:
Just FYI, the weld versions of the packaged imported below will have changed for cdi-3.0 and you don't want a cdi-2.0 dependency. I *really* want to try to find a way to avoid these imports all together.

Reply:
This is still part of the jsfContainer-2.3 feature, and  I haven't begun work on facesContainer-3.0 yet.  

After I added the Jakarta MyFace dependencies, one of the junit tests started failing in this bundle because it was looking Jakarta classes.  I needed to update this bnd file slightly to avoid the failure.